start/manage/fleets/edit.php

```diff
@@ -0,0 +1,74 @@
+<?php
+require '../ships/header.php';
+$shipId = isset($_GET['shipId']) ? $_GET['shipId'] : null;
+$fleetId = isset($_GET['fleetId']) ? $_GET['fleetId'] : null;
+$errors = [];
+
+use Service\Container;
+
+$container = new Container($configuration);
+$fleetLoader = $container->getFleetLoader();
+$fleetShip = $fleetLoader->getFleetShipByIds($shipId, $fleetId);
```

Comment:
I think you could find the Fleet object and find the Ship object.

Then add a helper method in the Fleet model to find the ShipFleet object:
```
public function findShipFleetByShip(AbstractShip $ship): ?ShipFleet
{
    foreach($this->shipFleets as $shipFleet) {
        if ($shipFleet->getShip()->getId() == $ship->getId()) {
            return $shipFleet;
        }
    }

    return null;
}
```